lib/renuo_bin_check/servant_thread.rb

```diff
@@ -8,6 +8,7 @@ def initialize(script_config)
       @script_config = script_config
       script_files = @script_config.script_files
       @cacher = Cacher.new(@script_config.script_name, script_files) if script_files
+      @result = nil
```

Comment:
i don't know if this is the ideal solution to fix https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md, but obviously the variable must exist due to reek.